dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -353,6 +354,8 @@ control dash_ingress(
 
         routing_action_apply.apply(hdr, meta);
 
+        tunnel_stage.apply(hdr, meta);
```

Comment:
do you mind to move the tunnel stage into routing_action_apply stage?

This helps maintaining the pipeline in mind model (once action_apply is done, packet transformation is done other than underlay routing).

Reply:
Done